src/BIP44Node.ts

```diff
@@ -404,6 +404,7 @@ function validateBIP44DerivationPath(
   path.forEach((nodeToken, index) => {
     const currentDepth = startingDepth + index;
 
+    // eslint-disable-next-line default-case
```

Comment:
What's the reasoning for changing this?

Reply:
I couldn't find a good way to cover the default case, and this would otherwise result in an uncovered line suddenly? I'm not sure how it was covered before.